src/combinatorics/Prob.lean

```diff
@@ -0,0 +1,157 @@
+/-
+Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+import tactic
+
+/-!
+# Probability on finsets
```

Comment:
For ease of reviewing, this file should probably be split into another PR.

Reply:
Agreed - I don't think this is ready for review yet!